terraform/README.md

```diff
@@ -1,62 +1,76 @@
 # Terraform Deployment
 
-We use terraform to create the spaces and services needed by the application and to deploy the application itself.
+We use terraform to create the spaces and services needed by the application and
+to deploy the application itself.
 
-We use terraform workspaces to keep the terraform state seperate for each environment.
+We use terraform workspaces to keep the terraform state seperate for each
+environment.
 
 We store our state in an s3 bucket on the PaaS in the terraform space.
 
 Deploys will happen automatically from Github Actions
 
 ## Manual Deployment
 
-We should avoid manual deployments but the following instructions will allow you to do one.
+We should avoid manual deployments but the following instructions will allow you
```

Comment:
Could we change this to account for the fact we have to manually deploy the training/pentest environment? Perhaps somehting like:

> We avoid manual deployments on production and staging, but require them on our training environment (pentest).